pkg/console/subresource/deployment/deployment.go

```diff
@@ -231,23 +233,38 @@ func GetLogLevelFlag(logLevel operatorv1.LogLevel) string {
 	return flag
 }
 
+func getStatusPageFlag(providers operatorv1.ConsoleProviders) string {
+	if providers.Statuspage != nil && len(providers.Statuspage.PageID) != 0 {
+		return fmt.Sprintf("--statuspage-id=%s", providers.Statuspage.PageID)
+	}
+	return ""
+}
+
 func consoleContainer(cr *operatorv1.Console, volConfigList []volumeConfig, proxyConfig *configv1.Proxy) corev1.Container {
 	volumeMounts := consoleVolumeMounts(volConfigList)
+	flags := []string{
```

Comment:
Want to do something like:

```go
flags := getFlags()
flags = withLogLevelFlag(cr.Spec, flags) // returns the flags unchanged if not set
flags = withStatusPageFlag(cr.Spec, flags) // returns the flags unchanged if not set
```
I think we could eliminate the extra `if len()` blocks then.